app/controllers/trainees/placements_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Trainees
+  class PlacementsController < BaseController
+    before_action :require_feature_flag
+
+    def new
+      @trainee = trainee
+      @placement_form = PlacementForm.new(trainee:)
+    end
+
+  private
+
+    def require_feature_flag
+      redirect_to(not_found_path) unless FeatureService.enabled?(:trainee_placement)
+    end
+  end
```

Comment:
hmmmm becoming a very common theme...

maybe 
in BaseController


```ruby
   def feature_flag = nil

    def require_feature_flag
      if feature_flag.present? && FeatureService.enabled?(feature_flag)
        redirect_to(not_found_path) 
      end
    end
```

in PlacementsController

```ruby
    before_action :require_feature_flag

    def feature_flag =  :trainee_placement
```

Reply:
@defong I've made the callback take a `feature_flag` param so that we can get away with just this in the controller:

```
    before_action { require_feature_flag(:trainee_placement) }
```